2006-04-03  Andreas Schwab  <schwab@suse.de>

        * sysdeps/unix/sysv/linux/powerpc/sys/procfs.h: Test for <asm/elf.h>
        with [! _ASM_POWERPC_ELF_H] as well as [!__PPC64_ELF_H].

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h,v
retrieving revision 1.13
retrieving revision 1.14
diff -u -r1.13 -r1.14
--- libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h	2006/01/13 07:58:57	1.13
+++ libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h	2006/04/04 06:58:58	1.14
@@ -35,7 +35,7 @@
 /* These definitions are normally provided by ucontext.h via 
    asm/sigcontext.h, asm/ptrace.h, and asm/elf.h.  Otherwise we define 
    them here.  */ 
-#ifndef __PPC64_ELF_H
+#if !defined __PPC64_ELF_H && !defined _ASM_POWERPC_ELF_H
 #define ELF_NGREG       48      /* includes nip, msr, lr, etc. */
 #define ELF_NFPREG      33      /* includes fpscr */
 #if __WORDSIZE == 32
